src/chainstate/stacks/db/blocks.rs

```diff
@@ -4113,7 +4113,42 @@ impl StacksChainState {
                     let event = STXEventType::STXMintEvent(STXMintEventData { recipient, amount });
                     events.push(StacksTransactionEvent::STXEvent(event));
                 }
-                Ok((total_minted, events))
+
+                let tx_version = if mainnet {
+                    TransactionVersion::Mainnet
+                } else {
+                    TransactionVersion::Testnet
+                };
+                let boot_code_address = boot_code_addr(mainnet);
+                let boot_code_auth = TransactionAuth::Standard(
+                    TransactionSpendingCondition::Singlesig(SinglesigSpendingCondition {
+                        signer: boot_code_address.bytes.clone(),
+                        hash_mode: SinglesigHashMode::P2PKH,
+                        key_encoding: TransactionPublicKeyEncoding::Uncompressed,
+                        nonce: 0,
+                        tx_fee: 0,
+                        signature: MessageSignature::empty(),
+                    }),
+                );
+
+                let lockup_tx = StacksTransaction::new(
+                    tx_version,
+                    boot_code_auth,
+                    TransactionPayload::ContractCall(TransactionContractCall {
+                        address: boot_code_address,
+                        contract_name: ContractName::try_from("lockup").unwrap(),
+                        function_name: ClarityName::try_from("get-lockups").unwrap(),
```

Comment:
Would "scheduled-unlock" be a better name for this phony contract call?

Reply:
Yeah, @zone117x to chime in and to confirm this, but I went with `get-lockups` because I think that the `stacks-blockchain-api` need to get a receipt associated with an existing method (and `get-lockups` is the only method in this contract).

Reply:
I don't think associating these events to this fake tx will work. It looks like the API won't be aware of this tx and lots of things will break. I think your suggestion in the comment below about attaching to the coinbase transaction makes a lot more sense.